graph/api/api.go

```diff
@@ -48,7 +48,7 @@ func graphNamespacesIstio(ctx context.Context, business *business.Layer, prom *p
 	// Create a 'global' object to store the business. Global only to the request.
 	globalInfo := graph.NewGlobalInfo(ctx, business, prom, config.Get())
 
-	trafficMap := istio.BuildNamespacesTrafficMap(ctx, o.TelemetryOptions, globalInfo)
```

Comment:
No longer passing context?

Reply:
context is actually in the globalInfo, and can be supplied from there.  So, the ctx param was redundant.